Tasks/MavenV2/task.json

```diff
@@ -103,6 +103,18 @@
             "helpMarkDown": "Provide a name for the test run.",
             "visibleRule": "publishJUnitResults = true"
         },
+        {
+            "name": "allowBrokenSymbolicLinks",
+            "aliases": [
+                "allowBrokenSymlinks"
+            ],
+            "type": "boolean",
+            "label": "Allow broken symbolic links",
+            "required": false,
+            "defaultValue": "true",
+            "groupName": "junitTestResults",
+            "helpMarkDown": "Set false to fail build when task finds broken symbolic link during publishing tests result"
```

Comment:
I think this option should only be shown when publishing test results is enabled since when it's disabled this option doesn't have any effect on the task

![image](https://user-images.githubusercontent.com/72494759/113271610-62ef1800-92e3-11eb-8b8b-3120e285670c.png)

```suggestion
            "helpMarkDown": "Set false to fail build when task finds broken symbolic link during publishing tests result",
            "visibleRule": "publishJUnitResults = true"
```

Reply:
Yes, sure. Thanks a lot.